lib/kitchen/directions/bake_numbered_table/v2.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeNumberedTable
+  class V2
+    def bake(table:, number:)
+      # The differences include:
+      # Adding a space to the end of the os-table class (to match original)
+      # Makes sure there's a caption span in every caption-container, whether there's a caption or no
+      # adds a possibility of a os-title span if one can be found within the caption
+      # adds the text from the os-title span
```

Comment:
Is this the best place to note differences?

Reply:
Maybe more in `main.rb` so someone looking at v1 and v2 could tell the diff?